integration-tests/features/steps/common.py

```diff
@@ -1117,6 +1117,55 @@ def stack_analysis_validate_alternate_components(context):
     perform_alternate_components_validation(json_data)
 
 
+def check_cve_value(cve):
+    pattern = "CVE-([0-9]{4})-[0-9]{4,}"
+
+    match = re.fullmatch(pattern, cve)
+    assert match is not None, "Improper CVE number %s" % cve
+
+    year = int(re.fullmatch(pattern, cve).group(1))
+    current_year = datetime.datetime.now().year
+
+    # well the lower limit is a bit arbitrary
+    assert year > 1980 and year <= current_year
+
+
+def check_cvss_value(cvss):
+    score = float(cvss)
+    # TODO: check the specificaion how to calculate the maximum possible value
+    # https://www.first.org/cvss/specification-document
+    assert score >= 0.0
```

Comment:
Ok, thanks @fridex. Let's use 10.0 (I just skimmed the specification and ask a friend from the Security Response Team and 10.0 is the right value!)